ksmutils/core.py

```diff
@@ -39,6 +39,7 @@ def __init__(
         # The amount we give the buyer if they win a dispute
         welfare_value: int = 1000000000,
```

Comment:
Is there anywhere else we can keep this? `welfare_value` seems so out of order in the Kusama class

Reply:
If it's just a constant used in one function, make it a constant in the method itself